test/RunSpec.hs

```diff
@@ -150,3 +138,18 @@ spec = do
           , "  ghc-options: -Wall -Werror"
           , "  default-language: Haskell2010"
           ]
+
+    it "renders source-repository without subdir correctly" $ do
+      renderSourceRepository (GithubConfig "https://github.com/hspec/hspec" Nothing)
+        `shouldBe` unlines [ "source-repository head"
+                           , "  type: git"
+                           , "  location: https://github.com/hspec/hspec"
+                           ]
```

Comment:
My reason is that if you change this code to e.g. `(T.pack . unlines) [...` later on, you have to realign the whole list.